tasks/release.py

```diff
@@ -133,6 +133,8 @@ def _is_version_higher(version_1, version_2):
         return False
     if version_1["patch"] < version_2["patch"]:
         return False
+    if version_1["rc"] == 0:
```

Comment:
Is it possible that the version_2 is an rc but not the version_1?
If so, we should add the edge case for version_2